src/Compilers/Test/Core/Compilation/MetadataReferenceExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using Roslyn.Test.Utilities;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities
+{
+    public static class MetadataReferenceExtensions
+    {
+        public static Guid GetModuleVersionId(this MetadataReference metadataReference)
+            => GetManifestModuleMetadata(metadataReference).GetModuleVersionId();
+
+        public static Guid GetModuleVersionId(this PortableExecutableReference peReference)
```

Comment:
>GetModuleVersionId [](start = 27, length = 18)

Is this method needed? It looks like the method above is sufficient.

Reply:
Good catch. Dleeted.